src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -246,48 +247,61 @@ def _accumulate_energy(self, area, child, is_cep: bool):
                     if trade.buyer is child.name:
                         self._total_energy += trade.offer.energy
 
-    def _export_house_pv_self_consumption(self, area):
-        house_pv_device = list()
-        house_load_device = list()
-        trade_by_pv = 0
-        total_energy_bought = 0
-        traded_from_pv = 0
-        if not _is_house_node(area):
-            return
+    def _export_area_sufficiency(self, area):
```

Comment:
How about to rename it to `_export_area_sufficiency_consumption` because it returns both ?

Reply:
Or how about _export_area_performance_index